jose-util/main.go

```diff
@@ -174,6 +174,39 @@ func main() {
 			},
 		},
 		{
+			Name:  "verify",
+			Usage: "verify a signature",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "key, k",
+					Usage: "Path to key file (PEM/DER)",
+				},
+				cli.StringFlag{
+					Name:  "input, in",
+					Usage: "Path to input file (stdin if missing)",
+				},
+				cli.StringFlag{
+					Name:  "output, out",
+					Usage: "Path to output file (stdout if missing)",
+				},
+			},
+			Action: func(c *cli.Context) {
+				keyBytes, err := ioutil.ReadFile(requiredFlag(c, "key"))
+				exitOnError(err, "unable to read key file")
+
+				verificationKey, err := jose.LoadPublicKey(keyBytes)
+				exitOnError(err, "unable to read private key")
+
+				obj, err := jose.ParseSigned(string(readInput(c.String("input"))))
```

Comment:
Does this accept the compact or full serialization? It might be nice to document which in the CLI flag description.